bazel/envoy_select.bzl

```diff
@@ -87,6 +87,13 @@ def envoy_select_enable_http3(xs, repository = ""):
         "//conditions:default": xs,
     })
 
+# Selects the given values if http datagram is enabled in the current build.
```

Comment:
@RyanTheOptimist should this be "if http datagrams are" rather than "http datagram is"?  I guess alternately we could say http datagram support.

If I'm correct on grammar we should update the select name to datagrams/_support

Reply:
Oh, yes. Good point. "if HTTP datagram support is enabled" would be a good way to phrase it.

And below I would name the function `envoy_select_enable_http_datagrams` (or `envoy_select_enable_http_datagram_support`).